visualization/src/pcl_visualizer.cpp

```diff
@@ -1422,6 +1423,30 @@ pcl::visualization::PCLVisualizer::setPointCloudRenderingProperties (
       actor->Modified ();
       break;
     }
+    case PCL_VISUALIZER_LUT:
+    {
+      // Check if the mapper has scalars
+      if (!actor->GetMapper ()->GetInput ()->GetPointData ()->GetScalars ())
+        break;
+      
+      // Check that scalars are not unisgned char (i.e. check if a LUT is used to colormap scalars assuming vtk ColorMode is Default)
+      if (actor->GetMapper ()->GetInput ()->GetPointData ()->GetScalars () ->IsA("vtkUnsignedCharArray"))
+        break;
+
+      // Get range limits from existing LUT
+      double *range;
+      range = actor->GetMapper ()->GetLookupTable ()->GetRange ();
+      
+      actor->GetMapper ()->ScalarVisibilityOn ();
+      actor->GetMapper ()->SetScalarRange (range[0], range[1]);
+      vtkSmartPointer<vtkLookupTable> table;
+      if (!pcl::visualization::getColormapLUT(int(value), table))
```

Comment:
Will need to `static_cast` to `LookUpTableRepresentationProperties` here. Also space is missing